cli: cdktf cli references manifest from cdktf

<!--- Please keep this note for the community --->

### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
- Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
- If you are interested in working on this issue or have submitted a pull request, please leave a comment

<!--- Thank you for keeping this note for the community --->

### cdktf & Language Versions

<!--- Please make note of the version you are using for the `hashicorp/terraform-cdk` and language bindings you are using.
If you are not running the latest version, please upgrade because your issue may have already been fixed. --->

- node v14
- cdktf-cli 0.6.1 (globally installed)

### Affected Resource(s)

- cdktf get

<!--- Please list the affected resources. --->

### Debug Output

```
< cdktf get
internal/modules/cjs/loader.js:892
  throw err;
  ^

Error: Cannot find module '../../cdktf/lib/manifest'
Require stack:
```

can be workaround by installing cdktf globally as well, but that's not expected

--- a/packages/cdktf-cli/lib/util.ts
+++ b/packages/cdktf-cli/lib/util.ts
@@ -4,7 +4,7 @@ import { https, http } from "follow-redirects";
 import * as os from "os";
 import * as path from "path";
 import { processLoggerError, processLoggerDebug } from "./logging";
-import { IManifest, Manifest } from "../../cdktf/lib/manifest";
+import { IManifest, Manifest } from "cdktf/lib/manifest";
 import { readConfigSync } from "./config";
 
 export async function shell(

